mintpy/objects/resample.py

```diff
@@ -338,13 +338,10 @@ def mark_lat_lon_anomoly(lat, lon):
 
             # parameter 1 - lalo_step (output grid)
             if self.lalo_step is None:
-                try:
-                    lat_c = (src_lat0 + src_lat1) / 2.
-                    lat_step, lon_step = ut.auto_lat_lon_step_size(self.src_meta, lat_c)
-                except AttributeError:
```

Comment:
We should keep the `try/except` logic, in case neither lut_meta nor src_meta has the metadata needed for `ut.auto_lat_lon_step_size()`, if `except KeyError:` still works as you described in #736.